### PR TITLE
Fix: Removed commented out line in pyproject.toml and setup project in local

### LINKED ISSUE #52: [C4GT Community]: Remove commented out line in pyproject.toml and setup project in your local
## Description
Project setup has become easier with steps outlined in [CONTRIBUTING](https://github.com/Xcov19/project-healthcare/blob/main/CONTRIBUTING.md)

There is no need for commented out dev groups in pyporject.toml

## Goals
- [ ] Remove the commented out line
- [ ] Ensure project is setup and runs successfully on your local

See [CONTRIBUTING](https://github.com/Xcov19/project-healthcare/blob/main/CONTRIBUTING.md)

### Expected Outcome

Project is setup and runs locally

### Acceptance Criteria

_No response_

### Implementation Details

Remove the commented out line

### Mockups/Wireframes

See: https://github.com/Xcov19/project-healthcare/issues/26#issuecomment-2315647630

### Product Name

project-healtlhcare

### Organisation Name

XCoV19

### Domain

Healthcare

### Tech Skills Needed

Python

### Mentor(s)

@codecakes 

### Complexity

Low

### Category

Backend, Beginner Friendly, Documentation, Maintenance, Testing

### PR DESCRIPTION
**Pull request by JiyaGupta-cs**
## Description

- Remove the commented out line
- Ensure project is setup and runs successfully on your local

I have uncommented the line instead of removing that to install them as dev dependencies

Fixes #52

### CODE REVIEW

**Comment by JiyaGupta-cs** [comment]
@codecakes Please review and merge it

File: pyproject.toml [hunk 1]
```diff
@@ -20,7 +20,7 @@ alembic = "^1.13.2"
 aiosqlite = "^0.20.0"
 sqlmodel = {version="^0.0.22"}
 
-# [tool.poetry.group.dev.dependencies]
+[tool.poetry.group.dev.dependencies]
```
Comment (codecakes) [inline pyproject.toml:23]: packages aren't grouped as dev dependencies anymore.
```suggestion
```

**Review by codecakes — CHANGES_REQUESTED** [review]

**Comment by JiyaGupta-cs** [comment]
@codecakes I have made changes, please review it

**Review by codecakes — APPROVED** [review]

**Comment by codecakes** [comment]
> @codecakes I have made changes, please review it

@JiyaGupta-cs resolve all review conversations

File: pyproject.toml (hunk 1 above)
Comment (JiyaGupta-cs) [reply pyproject.toml]: I have removed this line now as mentioned

**Comment by JiyaGupta-cs** [comment]
@codecakes I have resolved the review conversation, please merge it

**Review by codecakes — APPROVED** [review]

**Comment by VedantKhairnar** [comment]
Hello @JiyaGupta-cs ,
Thanks for your contribution.
As part of Augtoberfest, we want to reward you for your contributions, so pls fill out [this form](https://forms.gle/VBDUPWMYcZEbmVrQ9) by 19th Sept.
Thanks.

**Pull request merged**